src/sentry/static/sentry/app/routes.jsx

```diff
@@ -103,6 +103,7 @@ import TeamCreate from './views/teamCreate';
 import TeamDetails from './views/teamDetails';
 import TeamMembers from './views/teamMembers';
 import TeamSettings from './views/teamSettings';
+import TeamProjects from './views/settings/team/teamProjects';
```

Comment:
should i move this to a view?

Reply:
I would code split this

Reply:
probably skipping if that's ok, but i can revisit

Reply:
why skip?

Reply:
because i don't know how 😶 
is it as easy as wrapping in a lazyLoad? I could read code and figure it out but want to get started on mentions :/

Reply:
Don't worry about it then